languages/scala-2/template/src/main/scala/com/algorithmia/Algorithm.scala

```diff
@@ -0,0 +1,26 @@
+package com.algorithmia
+
+import com.algorithmia.handler.AbstractAlgorithm
+
+import scala.util.{Success, Try}
+
+class Algorithm extends AbstractAlgorithm[String, String]{
+
+  var someVariable: String = _
```

Comment:
This is a little odd. var is usually an anti-pattern in scala, although I understand you're trying to demonstrate loading. and assigning `_` seems weird... maybe it should be an `Option[String]` and assign `None`?